### ISSUE TITLE
Bridge does not work with openid-connect login

### ISSUE THREAD

**Issue by danastasio** [matrix-hacks/matrix-puppet-facebook#72]
So I recently enabled openid-connect logins on my homeserver, and I can't get my bridge to play nice with it. I've updated all the config files to refer to the new userpart, but I'm getting the below error:

`Error in handleThirdPartyRoomMessage [M_FORBIDDEN: User @<user>:<server> not in room !MesYDQBGQSnwjrXohm:matrix.davidanastasio.com (None)] {
  errcode: 'M_FORBIDDEN',
  name: 'M_FORBIDDEN',
  message: 'User @<user>:<server> not in room !MesYDQBGQSnwjrXoh<user>:<server> (None)',
  data: {
    errcode: 'M_FORBIDDEN',
    error: 'User @<user>:<server> not in room !MesYDQBGQSnwjrXohm:<server> (None)'
  },
  httpStatus: 403,
  event: MatrixEvent {
    event: {
      event_id: '~!MesYDQBGQSnwjrXohm:<server>:m1593378499008.4',
      user_id: '@<user>:<server>',
      room_id: '!MesYDQBGQSnwjrXohm:<server>',
      type: 'm.room.message',
      origin_server_ts: 1593378499008,
      content: {
        body: '<message_body> ﻿',
        msgtype: 'm.text'
      }
    },
    sender: null,
    target: null,
    status: 'not_sent',
    error: [Circular],
    forwardLooking: true,
    _pushActions: null,
    _replacingEvent: null,
    _clearEvent: {},
    _senderCurve25519Key: null,
    _claimedEd25519Key: null,
    _forwardingCurve25519KeyChain: [ [length]: 0 ],
    _decryptionPromise: null,
    _retryDecryption: false,
    _txnId: 'm1593378499008.4',
    _events: undefined,
    _eventsCount: 0,
    _maxListeners: undefined,
    [Symbol(kCapture)]: false
  }
} {
  roomId: '[redacted]',
  senderId: undefined,
  text: '<text_body>'
} ﻿`

Any ideas? The signal bridge doesn't work either. I'm turning off openid-connect until this can be resolved, but hopefully it's something simple.

**Comment by danastasio** [comment]
So it looks like the bridge is trying to use the old room, with the non-openid username, and not inviting the new user to a new room. Is there any way to reset the rooms that the bridge uses?

**Comment by danastasio** [comment]
I was able to resolve this. I needed to stop and delete the entire bridge, then run the following queries:

delete from room_aliases where room_alias like '%facebook%'
delete from room_memberships where room_id in ( select room_id from room_memberships where user_id like '%facebook%' );
delete from access_tokens where user_id like '%facebook%'

This reset everything and I'm now back up and running. If there's a better way to handle this I'm all ears.
